ext/em.cpp

```diff
@@ -1045,6 +1051,7 @@ void EventMachine_t::_CleanBadDescriptors()
 		rb_fd_set(sd, &fds);
 
 		int ret = rb_fd_select(sd + 1, &fds, NULL, NULL, &tv);
+		rb_fd_term(&fds);
```

Comment:
This codepath is only exercised when a BADF is encountered via the select() reactor. It is quite rare, so I left the heap allocation in place and am explicitly cleaning up here.